tiingo/wsclient.py

```diff
@@ -0,0 +1,93 @@
+import os
+import websocket
+try:
```

Comment:
I was wondering, what is this try-except import for?

Reply:
to import the correct version of ```thread``` regardless of python version